lib/matplotlib/figure.py

```diff
@@ -1503,10 +1477,8 @@ def gca(self, **kwargs):
                 "new axes with default keyword arguments. To create a new "
                 "axes with non-default arguments, use plt.axes() or "
                 "plt.subplot().")
-        if self._axstack.empty():
-            return self.add_subplot(1, 1, 1, **kwargs)
-        else:
-            return self._axstack()
+        current = self._axstack.current()
+        return current if current is not None else self.add_subplot(**kwargs)
```

Comment:
Minor style:

```suggestion
        ax = self._axstack.current()
        return ax if ax is not None else self.add_subplot(**kwargs)
```
I would have used `current_ax` but that would exceed the line length. When choosing between `current` and `ax` I find it more helpful to know that this is an axes. l.1480 already says that we're using the current thing.